CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
I think `OFF` makes sense, but for consistency's sake I think we should stick with `FALSE`. I'll default it to that and make sure it propagates as `FALSE` through `build-script`